catalog.c

```diff
@@ -266,7 +266,7 @@ catalog_get_backup_list(const pgBackupRange *range)
  * Find the last completed database backup from the backup list.
  */
 pgBackup *
-catalog_get_last_data_backup(parray *backup_list)
+catalog_get_last_data_backup(parray *backup_list, bool isFull)
```

Comment:
If the parameter `isFull` is always true in the current usage, let's just rename the function to `catalog_get_last_full_backup()`.